test-integration/utils/assertions/loans.js

```diff
@@ -1,45 +1,54 @@
 const assert = require('assert');
-const BigNumber = require("bignumber.js");
-const loanStatus = require("../../../test/utils/loanStatus");
-const { loans: loansActions } = require('../actions');
 const {
   NULL_ADDRESS,
-  ONE_DAY,
-  toBytes32,
-  toUnits,
 } = require("../../../test/utils/consts");
 
-const assertClosedLoan = async (
+const assertLoanValues = async (
   {loans},
   {testContext},
   {
     id,
-    status = loanStatus.Closed,
-    collateral = BigNumber('0'),
-    principalOwed = BigNumber('0'),
-    interestOwed = BigNumber('0'),
-    liquidated = false,
-    escrowTotalValueEth = BigNumber('0'),
-    escrowTotalValueToken = BigNumber('0'),
+    status,
+    collateral,
+    principalOwed,
+    interestOwed,
+    liquidated,
+    escrowTotalValueEth,
```

Comment:
the bad side of this is if I don't send any parameter, it will pass all the assertions.

Reply:
I think it should be strict on the values that you are asserting that the loan should have tho. Otherwise you could call this at any point for any loanId and just it would assume that the loan is closed. If you want to check that the loan has a certain status, you should specify which status you are asserting the loan to be in.